src/test/scala/modbat/mbt/ModbatTestHarness.scala

```diff
@@ -45,7 +70,7 @@ object ModbatTestHarness {
           current_iterator2 = it2.next
           if (current_iterator != current_iterator2){
             current_comparison=false
-            current_iterator="iterators different : \n validated_out_lines = " + current_iterator + "\n out = " + current_iterator2
+            current_iterator="iterators different : \n validated_out_lines = \n" + current_iterator + "\n out_filtered = \n" + current_iterator2
```

Comment:
Print the message here, so you can still access the contents of both iterators, as well as the content of the previous line that matched.